### PR TITLE
Added optional args: linewidth, linestyle, alpha

### PR DESCRIPTION
**Pull request by sid11k**
Tried to add the arguments to lineGraph method as mentioned in issue #1

### CODE REVIEW

File: tradingti/utils/_plot.py [hunk 1]
```diff
@@ -22,7 +22,8 @@
 
 
 def lineGraph(data, title = 'Untitled Graph', x_label = 'Date', y_label = 'Price', 
-    colors_palette = C_DEFAULT_COLORS_PALETTE):
+    colors_palette = C_DEFAULT_COLORS_PALETTE, linewidth = None * data.shape[1], linestyle = None * data.shape[1], 
+    alpha = None * data.shape[1]):
```
Comment (vsaveris) [inline tradingti/utils/_plot.py]: This it won't work I think. `data` is undefined at the moment were the last three arguments are evaluated. Additionally None*integer is unsupported operand type. I think an initialization can be: 
```
linewidth = [None], linestyle = [None], alpha = [None]
```
then below in the plt.plot you can do for all the three new arguments something similar to the color palette. For example for the linewidth:

`linewidth= linewidth[i % len(linewidth)]`

Which goes through the contents of the list.

File: tradingti/utils/_plot.py [hunk 2]
```diff
@@ -45,7 +46,13 @@ def lineGraph(data, title = 'Untitled Graph', x_label = 'Date', y_label = 'Price
...
             are more than the colors, then the list is scanned again from the zero
             index.
-
+        
+        linewidth (list of data.shape[1] members):
+        
+        linestyle (list of data.shape[1] members):
+        
+        alpha (list of data.shape[1] members):
+
```
Comment (vsaveris) [inline tradingti/utils/_plot.py]: Right. In the description of each you can mention a reference to matplotlib about the supported values. For example:

> linewidth (list of floats): _some description here_
> linestyle (list of strings): List of Matplotlib linestyles. See https://matplotlib.org/gallery/lines_bars_and_markers/linestyles.html  for possible values.
> alpha (list of floats) : _some description here_

File: tradingti/utils/_plot.py [hunk 3]
```diff
@@ -22,7 +22,7 @@
 
 
 def lineGraph(data, title = 'Untitled Graph', x_label = 'Date', y_label = 'Price', 
-    colors_palette = C_DEFAULT_COLORS_PALETTE):
+    colors_palette = C_DEFAULT_COLORS_PALETTE, linewidth = [None] * data.shape[1], linestyle = [None] * data.shape[1], alpha = [None] * data.shape[1]):
```
Comment (vsaveris) [inline tradingti/utils/_plot.py]: Multiplication with data.shape[1] it won't work, as data is unknown when arguments are evaluated, But also iit is not needed, due to the values rotation we plan to implement.

File: tradingti/utils/_plot.py (hunk 1 above)
Comment (sid11k) [reply tradingti/utils/_plot.py]: Okay! Thanks for the pointer

File: tradingti/utils/_plot.py (hunk 3 above)
Comment (sid11k) [reply tradingti/utils/_plot.py]: Sorry for all the commits

**Comment by vsaveris** [comment]
Nice, well done. I will test the code tomorrow and if all ok I will merge your commit.

**Pull request merged**